.gitignore

```diff
@@ -2,3 +2,6 @@
 *pyc
 *_flymake.py
 */*_flymake.py
+/build/
+/lib/
+/scripts-3.8/
```

Comment:
Is this python version dependent  - should it be `/scripts-3.*/` ?

Reply:
Indeed, distutils target directory for scripts.